storage/storage.go

```diff
@@ -35,3 +52,14 @@ func (f Fanout) Append(s *model.Sample) {
 		a.Append(s)
 	}
 }
+
+// Throttled returns true if at least one of the SampleAppenders in the Fanout
+// slice is throttled.
+func (f Fanout) Throttled() bool {
```

Comment:
We can throttle if a single one needs it – but that doesn't mean we have to throttle all the other ones.

It would be a bunch easier if `Append()` accepted a vector of samples.